events/calendars/views.py

```diff
@@ -0,0 +1,23 @@
+from django.shortcuts import render
+from django.views.generic.base import View
+from django.http import JsonResponse
+import json
+from event.models import User, EventUserAssignment, Event
+
+# Create your views here.
+class Calendar(View):
+    """
+    View used .
+    """
+    def get(self, request, user_id=None):
+        if not user_id:
+            events = Event.objects.all()
```

Comment:
You need to get only events from specific account (you need to get account from user)